springwolf-core/src/test/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/ChannelMergerTest.java

```diff
@@ -129,28 +130,28 @@ void shouldMergeDifferentMessagesForSameChannel() {
         // given
         String channelName = "channel";
         MessageObject message1 = MessageObject.builder()
-                .messageId("message1")
+                .messageId("messageString")
```

Comment:
marker: id vs name

Reply:
`messageId` is intended to be used as an `id` on the messages Map references, while the `name` is a 'machine-friendly name'. In most places I used the new `messageId` while in others I used the `name` to keep backward compatibility.

I think it's totally fair to provide a guideline and align the implementation to follow a single approach.

If we consider `name` as 'machine-friendly' it can totally work as a `messageId` and merge both concepts. I do not have a hard opinion there.

Reply:
Thanks. I had a closer look. messageId is used only for the spec.
I sticked to your suggestion to stay backwards compatible and use `name`